test/OpenTelemetry.Tests/Logs/LoggerProviderBuilderExtensionsTests.cs

```diff
@@ -42,6 +44,23 @@ public void LoggerProviderBuilderAddInstrumentationTest()
         Assert.True(((CustomInstrumentation)instrumentation[2]).Disposed);
     }
 
+    [Fact]
+    public void LoggerProviderBuilderAddInstrumentationLogsCorrectlyTest()
```

Comment:
What means _correctly_ here?

```suggestion
    public void LoggerProviderBuilderAddInstrumentationLogsTest()
```